### ISSUE TITLE
Any fix for log4j vulnerability?

### ISSUE THREAD

**Issue by isarwarfp** [mlangc/zio-interop-log4j2#1]
would there be an easy fix for updating log4j-core to fix log4j vulnerability.

**Comment by mlangc** [comment]
@imranbutt-java the library should work with newer log4j2 versions without any changes. Feel free to submit a PR that updates the dependency to log4j2. I'll be happy to review & merge it as well as publishing a new release.

**Comment by imranbutt-java** [comment]
Thanks @mlangc, I have updated log4j version to [2.17.1](https://logging.apache.org/log4j/2.x/maven-artifacts.html) but push get failed due to permission
```bash
Push failed remote: Permission to mlangc/zio-interop-log4j2.git 
denied to imranbutt-java. unable to access 'https://github.com/mlangc/zio-interop-log4j2.git/': 
The requested URL returned error: 403
```

**Comment by mlangc** [comment]
1.0.1 with an updated dependency has been released and is available on maven central. @imranbutt-java Thanks a lot for your contribution!

**Comment by jeejeeone** [comment]
> 1.0.1 with an updated dependency has been released and is available on maven central. @imranbutt-java Thanks a lot for your contribution!

1.0.1 released only for scala 2.13 in mvn central. Should there be releases for 2.12 (and 2.11) ?

**Comment by mlangc** [comment]
@jeejeeone sorry, I completely forgot about older Scala versions, but I pushed them now. Artifacts for Scala-2.11 and Scala-2.12 should be available in a few hours.
